frames/src/main/java/com/checkout/frames/cvvcomponent/models/CVVComponentConfig.kt

```diff
@@ -0,0 +1,16 @@
+package com.checkout.frames.cvvcomponent.models
+
+import com.checkout.base.model.CardScheme
+import com.checkout.frames.style.component.base.InputFieldStyle
+
+/**
+ * @param cardScheme - to validate entered CVV is valid or not in the SDK
+ * @param onCVVValueChange -  provides the isEnteredCVVValid for the given card scheme.
+ * So,based on isEnteredCVVValid value developer can decide to hit the tokenization for CVV component
+ * @param cvvInputFieldStyle - [InputFieldStyle] represent the input-field style for CVV Component
+ */
+public data class CVVComponentConfig(
```

Comment:
Let's add a sentence comment for `CVVComponentConfig`? like what we did for other data class